contracts/drafts/issuance/README.md

```diff
@@ -9,13 +9,16 @@ This is an Ethereum project that implements a simple Issuance that can be used f
 This issuance contract accepts investments using an accepted ERC20 token, and it will return to the investor a different ERC20 token if certain conditions are met.
 
 The issuance is governed by the following parameters:
-`currencyToken`: The token that is accepted in payment for investments.
-`issuanceToken`: The token that will be issued if conditions are met.
-`openingDate`: The time at which investments start being accepted.
-`closingDate`: The time at which investments stop being accepted, and distribution of issued tokens is possible.
-`issuePrice`:  The amount of currency tokens that are required to buy one issued token.
-`softCap`: The minimum mount of currency that needs to be raised for tokens to be distributed.
-`minInvestment`: The minimum amount of currency that can be invested in one go.
+* `currencyToken`: The token that is accepted in payment for investments.
+* `issuanceToken`: The token that will be issued if conditions are met.
+* `openingDate`: The time at which investments start being accepted.
+* `closingDate`: The time at which investments stop being accepted, and distribution of issued tokens is possible.
+* `issuePrice`:  The amount of currency tokens that are required to buy one issued token. The `issuePrice` parameter works like this:
+    - issuePrice > 0 : issuanceToken.mintAmount = currencyToken.investedAmount / issuePrice;
+    - issuePrice < 0 : issuanceToken.mintAmount = currencyToken.investedAmount * (-1) * issuePrice;
+    - issuePrice = 0 : revert.
```

Comment:
Hold on, what does a negative `issuePrice` do?

Reply:
Well, I thought about what you said:
>  2. Say that tokens should have 18 decimals, but prices are expressed with just 2 decimals. Then you can work out your own fixed point math div function.
> 3. Saying that prices should be expressed in full tokens is not great.

So, I decided that since Fixidity uses int256 to also have issuePrice as int256, and then, I could make use of the int properties and say:
1. If issuePrice is positive, you get a division (i.e. subunitary rate).
2. If issuePrice is negative, you get a multiplication (i.e. supraunitary rate).

Reply:
You will have to excuse most people getting confused with that.

The `issuePrice` should be denominated in currencyToken **wei**, and should be the amount of currencyToken wei that is needed to issue one **token** of the issuanceToken. Fixidity should be used just to keep the comma in place.

So, if currencyToken has 2 decimals you could say that the `issuePrice` is 50 (half a currencyToken per issuanceToken), or you could say that the `issuePrice is 200 (two currencyTokens per issuanceToken).

You just need to make sure that the user knows that `issuePrice` is expressed in currencyToken **wei** per one **issuanceToken** token.

Then:
 - Convert `currencyToken.investedAmount` and `issuePrice` to fixed point numbers.
 - Operate.
 - Convert back from fixed point.

If `issuanceToken` has a different number of decimals than `currencyToken` you should take account of that when converting back.

Reply:
Right, that was my initial intention, but I thought your point no.3 from my quote eliminated that option for me. Anyway, if I do it like that then I have to enforce `issuanceToken` (and, here and there, `currencyToken`) to inherit from `ERC20Detailed` and also to make `VentureEth` inherit from `ERC20Detailed`, so do I have your green light to make those changes? Or perhaps, I should say in the `README.md` that `issuanceToken` (or `currencyToken`) should have only 18 decimals?

Reply:
I think it's fair to require ERC20Detailed here.